client/src/js/components/bhFiltersApplied.js

```diff
@@ -1,7 +1,29 @@
 angular.module('bhima.components')
 .component('bhFiltersApplied', {
   templateUrl : 'partials/templates/bhFiltersApplied.tmpl.html',
+  controller : bhFiltersAppliedController,
   bindings: {
-    filters: '='
+    filters: '<',
+    onRemoveFilter: '&'  // fires to remove the filter
   }
 });
+
+bhFiltersAppliedController.$inject = [ '$scope', '$filter' ];
+
+function bhFiltersAppliedController($scope, $filter) {
```

Comment:
Could you leave a quick comment explaining what this directive does? 

Just to make sure I have it straight 
* Displays all filters passed in through the filters attribute
* Runs any Angular filter through the $filter service before displaying the value

A very short reasoning behind the $watch may be useful for the next developer working on filters.